src/main/java/teammates/common/datatransfer/FeedbackParticipantType.java

```diff
@@ -141,4 +141,25 @@ public String getDisplayNameGiver() {
     public String getDisplayNameRecipient() {
         return displayNameRecipient;
     }
+    
+    /**
+     * @return A user-friendly {@FeedbackParticipantType} list representing participant.
```

Comment:
- ~~user-friendly~~.
- `{@` is wrongly used.
- What does "representing participant" mean?

A better comment would be `a list of {@link FeedbackParticipantType} objects`. It is more concise and clear on what the method actually returns.